src/cmd/ksh93/include/variables.h

```diff
@@ -25,6 +25,15 @@
 #include        <option.h>
 #include        "FEATURE/options"
 #include        "FEATURE/dynamic"
+#include	<nval.h>
```

Comment:
This is a minor nit but shouldn't the Mamfile be updated to handle this `#include <nval.h>`?

<details open>
<summary>Mamfile patch</summary>

```diff
--- a/src/cmd/ksh93/Mamfile
+++ b/src/cmd/ksh93/Mamfile
@@ -311,6 +311,7 @@ make install
 						prev FEATURE/dynamic implicit
 						prev FEATURE/options implicit
 						prev ${PACKAGE_ast_INCLUDE}/option.h implicit
+						prev ${PACKAGE_ast_INCLUDE}/nval.h implicit
 					done include/variables.h
 					prev ${PACKAGE_ast_INCLUDE}/error.h implicit
 					prev ${PACKAGE_ast_INCLUDE}/stak.h implicit
```

</details>

Reply:
Yes, good point, forgot about that. But note there are two versions of nval.h. In this case it would depend on the nval.h in the ksh93 tree (`include/nval.h`), not the libast one.